tests/dialects/test_doris.py

```diff
@@ -19,3 +19,19 @@ def test_regex(self):
                 "doris": "SELECT REGEXP(abc, '%foo%')",
             },
         )
+
+    def test_trunc(self):
+        self.validate_all(
+            "SELECT TRUNC(sysdate)",
+            write={
+                "doris": "SELECT DATE(NOW())",
+            },
+        )
+
+    def test_trunc(self):
```

Comment:
can you please review your PRs before you submit them? this is an example of a low effort pr, you have a duplicate test_trunc here, 

delete both test_truncs and create

def test_doris:

and put all of these tests in there

Reply:
sorry, here is my problem